common/src/main/java/com/dumbdogdiner/stickyapi/annotation/DoNotCall.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.annotation;
+
+import java.lang.annotation.Documented;
+
+
+/**
+ * Do not call a method annotated with this, it will do bad things
+ */
+@Documented
+public @interface DoNotCall {
```

Comment:
Check where it's used

Reply:
Point still stands?

Reply:
I think its needed because if any of the methods are called, it will crash, so its for documentation.

Reply:
Would also like to add that if you dont want a specific method to be called, please use the correct modifier for your use case instead of creating annotations that wont prevent the user from executing this method.

Reply:
![image](https://user-images.githubusercontent.com/4931930/112874090-b9f4b300-907f-11eb-8cea-4668c4774287.png)

Reply:
I may be able to do stuff via refactor though

Reply:
Cloned from previous PR:
> Would prefer it if we can write things in a way where methods that are strictly internal can't be called externally.

So yes, would be nice if you can refactor this in a way that doesn't cause this issue.